python/orca/src/bigdl/orca/common.py

```diff
@@ -342,6 +342,8 @@ def stop_orca_context():
         ray_ctx = RayContext.get(initialize=False)
         if ray_ctx.initialized:
             ray_ctx.stop()
+        else:
+            RayContext._active_ray_context = None
```

Comment:
If changed to this logic, do we need to set active ray context to None when ray context is initialized?

Reply:
Do you mean set active ray context to None in ray_ctx.stop()?

Reply:
I mean remove the else, and set active to None anyway?